terraform/modules/services/airflow/dags/india/nwp-dag.py

```diff
@@ -22,7 +22,7 @@
 security_group = os.getenv("ECS_SECURITY_GROUP")
 cluster = f"india-ecs-cluster-development"
 
-with DAG('nwp-consumer', schedule_interval="10,25,40,55 * * * *", default_args=default_args, concurrency=10, max_active_tasks=10) as dag:
+with DAG('nwp-consumer', schedule_interval="5 0,6,12,18 * * *", default_args=default_args, concurrency=10, max_active_tasks=10) as dag:
```

Comment:
althought the times are right, are we sure that data comes in by 5 past. It noramlyl takes a few hours to run. I would b tempted to do `0 * * * *` so runs every hour

Reply:
Didn't Jacob say on a thread somewhere to run it 4 times a day? But you're right, looking at the bucket data it seems to be a good six hours after then init time that it comes through so far as I can tell

Reply:
yea it takes time to run, but we might just have to find out